source/geod24/LocalRest.d

```diff
@@ -1720,7 +1732,7 @@ unittest
 
             // Requests are dropped, so it times out
             assert(node.ping() == 42);
-            node.ctrl.sleep(10.msecs, true);
+            node.ctrl.sleep(20.msecs, true);
```

Comment:
Why was this value changed?

Reply:
The new PR could not process 500 in a second. So I set 100 in 3 seconds.